tutorial/index.jade

```diff
@@ -84,7 +84,7 @@
     :markdown
       ## Sending the Message
 
-      Here is the method that actually sends the sms. If we want to improve
+      Here is how we actually send the sms. If we want to improve
```

Comment:
@kathyguerron _sms_ is an acronym for _Short Message Service_. Could you please capitalise it?

_As a side note you should always capitalise acronyms._